src/Personal/personalStackNavigation.jsx

```diff
@@ -0,0 +1,23 @@
+import { createAppContainer } from 'react-navigation';
+import { createStackNavigator } from 'react-navigation-stack';
+import Index from './index';
+import Register from './register';
+
+const MainNavigator = createStackNavigator({
+  Index: {
+    screen: Index,
+    navigationOptions: {
+      header: null,
+    },
+  },
+  Register: {
+    screen: Register,
+    navigationOptions: {
+      header: null,
+    },
+  },
+});
+
+const personalNavigator = createAppContainer(MainNavigator);
```

Comment:
is it really needed to create an `AppContainer`?

Reply:
I tried without creating a AppContainer, but it will be failed.

Reply:
I tried without creating a AppContainer, but it will be failed.